### PR TITLE
Complete crates.io integration

### PR DESCRIPTION
**Pull request by calebbrown**
This change adds full support for crates.io for Package Analysis.

Currently the analysis worker supports crates.io, but is not yet receiving work from the scheduler.

This change adds support to the scheduler. It also adds support to allow any package on crates.io to be ingested using the analysis_runner.sh script

### CODE REVIEW

**Comment by calebbrown** [comment]
+ @another-rex

**Comment by calebbrown** [comment]
This change mostly completes #33. There is possibly some minor fixes needed for BigQuery imports.

File: cmd/scheduler/main.go [hunk 1]
```diff
@@ -24,14 +24,42 @@ const (
 	retryExpRate  = 1.5
 )
 
+type ManagerConfig struct {
+	// ExcludeVersions is a list of regexp expressions that will ignore a
+	// specific version of a package if it is matched by any of the expressions.
```
Comment (maxfisher-g) [inline cmd/scheduler/main.go:29]: The wording is a little ambiguous - I thought it might apply the rule to only certain specified packages.
Suggest rewording, e.g. "Versions of any package whose version string matches a regexp in this list will be ignored"

Comment (calebbrown) [reply cmd/scheduler/main.go]: Updated. PTAL

Comment (maxfisher-g) [reply cmd/scheduler/main.go]: Awesome, looks good!

**Review by maxfisher-g — APPROVED** [review]
